Plate_2_data/1.cellprofiler_quality_control/README.md

```diff
@@ -0,0 +1,20 @@
+# Create LoadData CSV and create maximum projection images 
+
+In this module, we create a LoadData csv and run a CellProfiler pipeline that performs maximum projection on the images and saves them into a new folder.
+This pipeline names the outputted max projected images as the last z-plane.
+In our case we have 4 z-planes (p01-p03) so our images contain the last z-plane p04.
+
+## pe2loaddata
+
+To create the LoadData csv file for CellProfiler, we use a software called [pe2loaddata](https://github.com/broadinstitute/pe2loaddata/tree/220ac512bfc0c2e582d379b19411c1585272aee3). 
+This will create a LoadData csv from the Phenix metadata XML file generated from data acquisition. 
```

Comment:
This section feels very similar to the section below called `Create the LoadData CSV and create max projection images`

Consider consolidating sections to increase clarity

Reply:
I agree that they sound very similar. I added more information on the software. See commit https://github.com/WayScience/pyroptosis_signature/pull/20/commits/0b6a5953d29cbd11eea74fc8d01ecbef570459ac